docs/ossec_alerts.rst

```diff
@@ -87,42 +87,55 @@ If you have your GPG public key handy, copy it to
 install\_files/ansible-base and then specify the filename in the
 ``ossec_alert_gpg_public_key`` line of prod-specific.yml.
 
-If you don't have your GPG key ready, you can run GnuPG on the command
-line in order to find, import and export your public key. It's best to
-copy the key from a trusted and verified source, but you can also
-request it from keyservers using the known longid. Looking it up by
-email address could cause you to obtain a wrong, malicious or expired
-key. Just replace the key ID in the following example with yours:
+If you don't have your GPG key ready, you can run GnuPG on the command line in
+order to find, import, and export your public key. It's best to copy the key
+from a trusted and verified source, but you can also request it from keyservers
+using the known fingerprint. Looking it up by email address or a shorter key ID
+format could cause you to obtain a wrong, malicious, or expired key. Instead, we
+recommend you type out your fingerprint in groups of four (just like GPG prints
+it) enclosed by parentheses.  The reason we suggest this formatting for the
+fingerprint is simply because it's easiest to type and verify correctly. In the
+code below simply replace ``<fingerprint>`` with your full, space-separated
+fingerprint:
 
-Download your key and import it into the local keyring:
+Download your key and import it into the local keyring: ::
 
-::
+    gpg --keyserver hkp://qdigse2yzvuglcix.onion --recv-key "<fingerprint>"
 
-    gpg --recv-keys 0xFD720AD9EBA34B1C
+Again, when passing the full public key fingerprint to the ``--recv-key`` command, GPG
+will implicitly verify that the fingerprint of the key received matches the
+argument passed.
 
-Export the key to a file:
+.. caution:: If GPG warns you that the fingerprint of the key received
+             does not match the one requested **do not** proceed with
+             the installation. If this happens, please email us at
+             securedrop@freedom.press.
 
-::
+Next we export the key to a local file. Here it is safe to use your email
+address associated with the key instead of your fingerprint since we are
+performing a local action and do not need to worry about network-based attacks
+as when fetching a key from a keyserver. Replace ``<email>`` with this email
+address below. ::
```

Comment:
Ok, that's true, but do we need to add this complexity to the docs? Why not just stay consistent and recommend they use the fingerprint for everything?